src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web.UnitTests/Services/ModeratorOutcomeOrchestrator/GetInModerationOutcomeReviewViewModelTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Common.Extensions;
+using SFA.DAS.AdminService.Common.Testing.MockedObjects;
+using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Apply;
+using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Moderator;
+using SFA.DAS.RoatpAssessor.Web.Infrastructure.ApiClients;
+using SFA.DAS.RoatpAssessor.Web.Models;
+using SFA.DAS.RoatpAssessor.Web.ViewModels;
+
+namespace SFA.DAS.RoatpAssessor.Web.UnitTests.Services.ModeratorOutcomeOrchestrator
+{
+    [TestFixture]
```

Comment:
I'm not sure what's going on here. It doesn't follow the pattern I used in AssessorOutcomeControllerTests